### PR TITLE
ACCT-40 authentication testing for auth0

### PR DESCRIPTION
**Pull request by shashi-kubra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,239 @@
...
+A template file named `cypress.env.dist` has been provided containing all the sensitive fields that must be populated to run the tests locally.
+This file should be renamed to `cypress.env.json` and populated with values from LastPass. This file is already in the GitIgnore file, so it will not show in your list of modified files.
+
+#### Example cypress.env.json - these values are examples. Please retrieve actual values from Lastpass.
+```json
+{
+	"auth_audience": "https://test.kubra.com/", // The audience for retrieving oauth token
+	"auth_url": "https://some.auth0.endpoint.com/oauth/token",  // The oauth/token endpoint
+	"auth_client_id": "xxxxxxxxxxxxxxxxxx", // The Client ID for retrieving oauth token
+	"auth_username": "test@kubra.com", // The username for retrieving oauth token
```
Comment (austinlaske-kubra) [inline README.md:118]: Is this actual credentials?

File: cypress/support/commands.js [hunk 2]
```diff
@@ -18,7 +27,7 @@ Cypress.Commands.add('loginWithAuth0', (overrides = {}) => {
     body: {
       grant_type: 'password',
       username: Cypress.env('auth_username'),
-      password: Cypress.env('auth_password'),
+      password: atob(Cypress.env('auth_password')),
```
Comment (austinlaske-kubra) [inline cypress/support/commands.js:30]: not sure what atob is?

File: cypress/tests/securityTests/authentication testing.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
+import { PostCall, namespacesEndpoint } from "../utils"
```
Comment (austinlaske-kubra) [inline cypress/tests/securityTests/authentication testing.js:1]: This file has a naming convention issue. Should be authenticationTesting.js or authentication_testing.js

File: cypress/tests/utils.js [hunk 4]
```diff
@@ -1,29 +1,78 @@
...
-            d2 = Math.floor(d2/16);
+            r = (d2 + r) % 16 | 0;
+            d2 = Math.floor(d2 / 16);
         }
         return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
     });
+}
+
+export const PostCall = (endpoint, _body, _auth =  { bearer: `${Cypress.env("DefaultAuth0Token")}` }) => {
+
```
Comment (austinlaske-kubra) [inline cypress/tests/utils.js:34]: Delete whitespace

**Review by amruthadivi-kubra — APPROVED** [review]

**Review by austinlaske-kubra — APPROVED** [review]

**Pull request merged**